rllib/tests/backward_compat/test_gym_env_apis.py

```diff
@@ -143,6 +189,55 @@ def test_():
             lambda: test_(),
         )
 
+    def test_multi_agent_gymnasium_old_api(self):
```

Comment:
Can you also add this test scenario?
If I am on the new gymnasium API, but my reset() throws an exception we should not try to auto-wrap it.

Reply:
added